ledger/spverificationtracker_test.go

```diff
@@ -38,7 +38,7 @@ const unusedByStateProofTracker = basics.Round(0)
 type StateProofTrackingLocation uint64
 
 const (
-	any StateProofTrackingLocation = iota
+	stateproofTrackingLoc StateProofTrackingLocation = iota
```

Comment:
spver is kind of common prefix here
```suggestion
	spverDBLoc StateProofTrackingLocation = iota
```